src/core/stores/VNodes/RangeNode.ts

```diff
@@ -0,0 +1,16 @@
+import { VNode, VNodeType } from '../VNode';
+
+export class RangeNode extends VNode {
+    properties = {
+        atomic: true,
+    };
+    constructor(tailOrHead: 'tail' | 'head') {
```

Comment:
`side` ?

Reply:
`edge` ?

Reply:
Or `type` and the expected value is either `VNodeType.Range_TAIL` or `VNodeType.RANGE_HEAD` to avoid creating a new string bridge to these values.